aksparse/client/aksparse/sparse.py

```diff
@@ -322,41 +342,22 @@ def dot(self, input):
         if str(type(input)) == "<class 'arkouda.pdarrayclass.pdarray'>":
             return self.vector_matrix_dot(input)
         if str(type(input)) == "<class ''>":
-            return self.matrix_matrix_dot(input)
+            raise NotImplementedError
+            # return self.matrix_matrix_dot(input)
```

Comment:
If the old code is not needed for something, we should remove the comment.

Reply:
Same thing, deleted!